modules/lymphgen/1.0/config/default.yaml

```diff
@@ -29,11 +64,16 @@ lcr-modules:
                 bcl6truevalues: "__UPDATE__"  # Can use a list
                 bcl6falsevalues: "__UPDATE__"
             lymphgen_input:
-                seq_type: "genome"  # Availible options: exome/genome (they do the same thing) and targeted (for panels)
+                seq_type: "__UPDATE__"  # Availible options: exome/genome (they do the same thing) and targeted (for panels)
                 use_log_ratio: "__UPDATE__"
             lymphgen_run:
                 lymphgen_path: "/projects/rmorin/software/lcr-scripts/LymphGen/Run_Lymphgen.R"
```

Comment:
Could you update this to the new path that the LymphGen module will be tracked at?